syntax = "proto3";
package RunSight.Shared.Protobuf.Responses;
option csharp_namespace = "RunSight.Shared.Protobuf.Responses";

import "Protobuf/Enums/RetCodes.proto";

message ConfigureUpdateRsp {
    Enums.RetCodes RetCode = 1;
    string Message = 2;
}
